test.ts

```diff
@@ -0,0 +1,134 @@
+import { which } from "./deps/deno_which.ts"
+import { unimplemented } from "./deps/std/assert.ts"
+import { parse as parseFlags } from "./deps/std/flags.ts"
+import { blue, dim, gray, green, red, yellow } from "./deps/std/fmt/colors.ts"
+import { walk } from "./deps/std/fs.ts"
+import { Buffer, readLines } from "./deps/std/io.ts"
+import * as path from "./deps/std/path.ts"
+import { readerFromStreamReader, writeAll } from "./deps/std/streams.ts"
+import { parse as parseFrontmatter } from "./frontmatter.ts"
+
+const { _, reload, ...rest } = parseFlags(Deno.args, {
+  alias: {
+    b: "browser",
+    c: "concurrency",
+    p: "project",
+    r: "reload",
+  },
+  string: ["browser", "concurrency", "project", "reload"],
+  default: {
+    concurrency: Infinity,
+  },
+})
+
+const include: string[] = []
+for await (
+  const { path: pathname } of walk(".", {
+    exts: [".ts", ".tsx"],
+    followSymlinks: true,
+    includeDirs: false,
+    match: _.map((value) => {
+      if (typeof value !== "string") {
+        throw new Error(
+          `Specified an invalid include \`${value}\` (expected a glob or path to example file)`,
+        )
+      }
+      return path.isGlob(value) ? path.globToRegExp(value) : new RegExp(value)
+    }),
+  })
+) include.push(pathname)
+
+const concurrency = +rest.concurrency
+
+const project = rest.project ?? await (async () => {
+  for (const pathname of ["deno.json", "deno.jsonc"]) {
+    try {
+      return JSON.parse(await Deno.readTextFile(pathname))
+    } catch (_e) {}
+  }
+  return
+})()
+const _importMapURL = project.importMap
+  ? path.toFileUrl(path.join(Deno.cwd(), project.importMap))
+  : undefined
+
+const browser = typeof rest.browser === "string"
+  ? rest.browser === "" ? await which("chromium") : rest.browser
+  : undefined
+if (browser === "") throw new Error(`Failed to detect chromium path. Specify via \`--browser\`.`)
```

Comment:
`which` returns `undefined` if it doesn't find a match, so this conditional will never be reached. I don't think we need to call `which` here; we can just default to `"chromium"` and if it's not in the path it'll error downstream.
```suggestion
const browser = rest.browser === undefined ? undefined : rest.browser || "chromium"
```